Packs/IvantiHeat/Integrations/IvantiHeat/IvantiHeat.py

```diff
@@ -0,0 +1,285 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+import json
+import requests
+import dateparser
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CONSTANTS '''
+TOKEN = demisto.params().get('token')
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+OBJECT_HEADERS = ['RecId', 'Subject', 'Name', 'Status', 'CreatedDateTime', 'Symptom', 'Urgency', 'OwnerTeam',
+                  'IncidentNumber', 'CreatedBy', 'Details', 'Owner', 'Category', 'Description', 'Priority',
+                  'Email', 'TypeOfIncident', 'ClosedDateTime', 'ActualCategory', 'SocialTextHeader']
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+    def do_request(self, method, url_suffix, params=None, data=None, json_data=None, resp_type='json', files=None):
+
+        res = self._http_request(method, url_suffix, params=params, data=data, json_data=json_data, files=files,
+                                 resp_type=resp_type, ok_codes=(200, 201, 204), return_empty_response=True)
+
+        if resp_type == 'other' or isinstance(res, dict) or isinstance(res, list):
+            return res
+        return {}
+
+
+def test_module(client):
```

Comment:
in case `fetch-incidents` is enabled, need to validate the fetch_time parameter as well.